doc/win32-usb-troubleshoot.md

```diff
@@ -0,0 +1,31 @@
+# Troubleshooting LIBUSB_* errors on win32 platforms
+
+nRF Connect Desktop uses [libusb](https://libusb.info/) and [node-usb](https://github.com/tessel/node-usb) to know how many USB-capable Nordic SoCs (e.g. nRF52840) are connected to your PC, and also to set them into DFU mode when they need to be re-programmed by a nRF Connect Desktop application.
+
+While this makes it easier to develop nRF Connect Desktop applications, this might cause some error to show up in some win32 environments. This document collects the known scenarios where that happens.
+
+## Windows 7 right after installation
+
+A `LIBUSB_ERROR_NOT_FOUND` error shows up when performing the following actions in this exact order:
+
+- Run the nRF Connect Desktop installer.
```

Comment:
Don't use period at end of bullet points for single sentences, be consistent.

Reply:
@bencefr consider using a more suggestive approach in code review comments. Probably not intended, but the language is a bit confrontational.